### PR TITLE
14.0 - MIG - sale order line sequence

### PR DESCRIPTION
**Pull request by patrickt-oforce**
Back port of module `sale_order_line_sequence` to v 14.0

--
Conferm have signed  CLA https://odoo-community.org/page/cla and have read the contributing guideline at https://odoo-community.org/page/contributing

### CODE REVIEW

**Comment by rousseldenis** [comment]
@patrickt-oforce Thanks for your contribution.

Can you follow migration guide https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-14.0

(The commands are quite the same - inversing branch names)

**Comment by patrickt-oforce** [comment]
> @patrickt-oforce Thanks for your contribution.
> 
> Can you follow migration guide https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-14.0
> 
> (The commands are quite the same - inversing branch names)
Hi @rousseldenis, sorry for my mistakes but i've followed literraly the guide you suggested me. What means with `inversing branch name`?

File: sale_order_line_sequence/__manifest__.py [hunk 1]
```diff
@@ -1,12 +1,16 @@
...
 
 {
     "name": "Sale Order Line Sequence",
     "summary": "Propagates SO line sequence to invoices and stock picking.",
-    "version": "15.0.1.0.0",
-    "author": "Eficent, Serpent CS, Odoo Community Association (OCA)",
+    "version": "14.0.1.0.0",
+    "author": "Eficent, "
+    "Serpent CS, "
+    "Openforce srls, "
```
Comment (rousseldenis) [inline sale_order_line_sequence/__manifest__.py]: Migration does not allow you to put yourself as author.

Instead put a line in readme/CONTRIBUTORS.rst

Comment (patrickt-oforce) [reply sale_order_line_sequence/__manifest__.py]: Ok sorry. I'm fix this immediatly

**Comment by rousseldenis** [comment]
> > @patrickt-oforce Thanks for your contribution.
> > Can you follow migration guide https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-14.0
> > (The commands are quite the same - inversing branch names)
> > Hi @rousseldenis, sorry for my mistakes but i've followed literraly the guide you suggested me. What means with `inversing branch name`?

Read especially the 'Full process' part. You need to have first a IMP commit with pre-commit changes before migration one.

image

File: sale_order_line_sequence/__manifest__.py (hunk 1 above)
Comment (patrickt-oforce) [reply sale_order_line_sequence/__manifest__.py]: @rousseldenis i need to remove the copyright line added?

**Comment by patrickt-oforce** [comment]
> > > @patrickt-oforce Thanks for your contribution.
> > > Can you follow migration guide https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-14.0
> > > (The commands are quite the same - inversing branch names)
> > > Hi @rousseldenis, sorry for my mistakes but i've followed literraly the guide you suggested me. What means with `inversing branch name`?
> 
> Read especially the 'Full process' part. You need to have first a IMP commit with pre-commit changes before migration one.
> 
> image

Thnak you for advice, can i rebase my history, having that as first commit?

**Comment by rousseldenis** [comment]
> > > > @patrickt-oforce Thanks for your contribution.
> > > > Can you follow migration guide https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-14.0
> > > > (The commands are quite the same - inversing branch names)
> > > > Hi @rousseldenis, sorry for my mistakes but i've followed literraly the guide you suggested me. What means with `inversing branch name`?
> > 
> > 
> > Read especially the 'Full process' part. You need to have first a IMP commit with pre-commit changes before migration one.
> > image
> 
> Thnak you for advice, can i rebase my history, having that as first commit?

No, let as is. For the next time :-)

**Comment by rousseldenis** [comment]
> > > > > @patrickt-oforce Thanks for your contribution.
> > > > > Can you follow migration guide https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-14.0
> > > > > (The commands are quite the same - inversing branch names)
> > > > > Hi @rousseldenis, sorry for my mistakes but i've followed literraly the guide you suggested me. What means with `inversing branch name`?
> > > 
> > > 
> > > Read especially the 'Full process' part. You need to have first a IMP commit with pre-commit changes before migration one.
> > > image
> > 
> > 
> > Thnak you for advice, can i rebase my history, having that as first commit?
> 
> No, let as is. For the next time :-)

That is done to separate changes that are real migration ones than cosmetics ones to easy review.

**Comment by patrickt-oforce** [comment]
> > > > > @patrickt-oforce Thanks for your contribution.
> > > > > Can you follow migration guide https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-14.0
> > > > > (The commands are quite the same - inversing branch names)
> > > > > Hi @rousseldenis, sorry for my mistakes but i've followed literraly the guide you suggested me. What means with `inversing branch name`?
> > > 
> > > 
> > > Read especially the 'Full process' part. You need to have first a IMP commit with pre-commit changes before migration one.
> > > image
> > 
> > 
> > Thnak you for advice, can i rebase my history, having that as first commit?
> 
> No, let as is. For the next time :-)

Thank you, but i've restarted migration as suggested

**Review by JuanyDForgeflow — APPROVED** [review]
Functional Review LGTM

**Review by AaronHForgeFlow — COMMENTED** [review]
Functional :+1:  Can you check travis? It is other module, maybe a rebase should suffice to fix it

**Comment by patrickt-oforce** [comment]
> Functional +1 Can you check travis? It is other module, maybe a rebase should suffice to fix it

Hi @AaronHForgeFlow, i've seen travis output, but the error is referred to a test failed in a module that i've not touched. My effort is to backport the module to a previous version. What do you mean with: "maybe a rebase should suffice to fix it"?

**Comment by AaronHForgeFlow** [comment]
@patrickt-oforce , I mean that maybe the root cause of the error is now fixed in the base branch. You can just rebase your branch and perhaps it will just pass the checks. I think it worth a try.

**Comment by patrickt-oforce** [comment]
> @patrickt-oforce , I mean that maybe the root cause of the error is now fixed in the base branch. You can just rebase your branch and perhaps it will just pass the checks. I think it worth a try.

Hi @AaronHForgeFlow , i've rebased the branch at actual 14.0 HEAD

**Comment by patrickt-oforce** [comment]
Need to rebase, last commit?

**Comment by AaronHForgeFlow** [comment]
@patrickt-oforce It it fine now I think. Tests are failing because of another module, "sale_by_packaging" I wonder if this module is not compatible with that one, or it is just an unrelated error :S

**Comment by MiquelRForgeFlow** [comment]
Superceded by https://github.com/OCA/sale-workflow/pull/2104.

**Comment by HaraldPanten** [comment]
Superseded by https://github.com/OCA/sale-workflow/pull/2104

**Pull request closed without merging**